dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/GraceFulShutDown.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.dubbo.rpc;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.dubbo.common.extension.SPI;
+
+/**
+ * this interface is used to enable grace shutdown
+ * any class implements this interface should be a spring bean or add spi extension config
+ * if you use spring ,ShutdownHookListener will use it and flush resources in ContextClosedEvent
+ * Without spring,please add a file under META-INFO/dubbo/internal which named org.apache.dubbo.rpc.GraceFulShutDown
+ * and then write graceful=xxx.xxx.xxx.Xxx which implement GraceFulShutDown into it
+ * application will use DubboShutdownHook to invoke afterRegistriesDestroyed and afterProtocolDestroyed
+ * to flush resources in closed stage
+ * @since 2.7.3
+ */
+@SPI
+public interface GraceFulShutDown {
```

Comment:
Graceful is a word.  So you can use it directly.   please GraceFul -> Graceful

Reply:
Ok,I will update it